content/guides/models/sweeps/add-w-and-b-to-your-code.md

```diff
@@ -219,36 +241,43 @@ def main():
 main()
 ```
 
-Navigate to your CLI. Within your CLI, set a maximum number of runs the sweep agent should try. This is step optional. In the following example we set the maximum number to five.
+Navigate to your CLI. Within your CLI, set a maximum number of runs the sweep
+agent should try. This is step optional. In the following example we set the
+maximum number to five.
 
 ```bash
 NUM=5
 ```
 
-Next, initialize the sweep with the [`wandb sweep`]({{< relref "/ref/cli/wandb-sweep.md" >}}) command. Provide the name of the YAML file. Optionally provide the name of the project for the project flag (`--project`):
+Next, initialize the sweep with the [`wandb
+sweep`]({{< relref "/ref/cli/wandb-sweep.md" >}}) command. Provide the name of
+the YAML file. Optionally provide the name of the project for the project flag
+(`--project`):
```

Comment:
```suggestion
Next, initialize the sweep with the [`wandb sweep`]({{< relref "/ref/cli/wandb-sweep.md" >}}) command. Provide the name of the YAML file. Optionally provide the name of the project for the project flag (`--project`):
```
For the future, Markdown diffs are much easier to follow and leave feedback on if you do not hard wrap the lines. If you must hard wrap, it's best to avoid splitting up the Markdown link text.